### PR TITLE
Feat: Allow to set filename for template (FIXED)

### PR DESCRIPTION
**Pull request by spectrewall**
Added filename parameter to allow developers to set filename in template message.

Work based on PR: https://github.com/netflie/laravel-notification-whatsapp/pull/8 but as it seens to be forgotten, i took the liberty to add what was asked.

### CODE REVIEW

File: src/Component/Document.php [hunk 1]
```diff
@@ -27,6 +32,7 @@ public function __construct(string $link)
         }
 
         $this->link = $link;
+        $this->filename = empty($filename) ? 'document' : $filename;
```
Comment (aalbarca) [inline src/Component/Document.php:35]: Is it necessary to provide a default file name?

Comment (spectrewall) [reply src/Component/Document.php:35]: Not totally, but in my tests, unamed files are send as "null.pdf", what made the file thumb not load in the whatsapp conversation. so call it "document.pdf" by default fixed this little problem (from the whatsapp api)

**Comment by aalbarca** [comment]
Hi @spectrewall this PR is blocked due an unsuccesful pipeline: https://github.styleci.io/analyses/yrmB3o?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

You can fix this error running php cs fixer or fix manually the style issues.

**Comment by spectrewall** [comment]
Hi @aalbarca, Sorry for being late on this, but a just fixed the error.

**Pull request merged**